src/ray/protobuf/core_worker.proto

```diff
@@ -424,6 +424,16 @@ message ReportGeneratorItemReturnsReply {
   int64 total_num_object_consumed = 1;
 }
 
+message CreateMutableObjectRequest {
```

Comment:
Can you rename this, since it is no longer "creating" the mutable object? Also please add docs to each protobuf field.